preprocessing/tabular_data/feature_selection_tabular_data.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Is there a reason for using the 3rd party versions of xgboost and sklearn?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Yes. Mainly because both libraries are commonly used and very lightweight making it easy to quickly train and test an ML model. </p>